packages/donations-app/app/components/my-impact/myImpactHomeView.html

```diff
@@ -15,12 +15,12 @@
     <div ng-if="dashCtrl.auth.isManager() && !dashCtrl.auth.isSuperadmin()" class="col-lg-12 col-md-12 dashboard-nav-top">
       <div class="dashboard-image col-lg-1 col-md-1">
         <img class="dashboard-image-in"
-             src="{{dashCtrl.charity.initializerImg}}"
+             src="{{dashCtrl.loggedUser.charityForCharityAdmin.darkPicture}}"
```

Comment:
This is still not clear for me, why should we put a charity logo on my-impact page which should be targeted at individual users?

Reply:
I thought isManager() checks for project Manager? - So it isn't seen by normal users?

Reply:
Let's leave it for now, I'll also talk to Areti. 
I'm afraid of confusing my-impact dashboard and charity-dashbord by charity admins